bmvc/grid_cell_path_integration_for_movement_based_visual_object_recognition/README.md

```diff
@@ -1,12 +1,12 @@
 Purpose of the code
 ===============
-The following implements a network using grid-cell computations that can perform basic object recognition on visual tasks such as the MNIST data-set. It builds on the architecture described in [Lewis et al 2019](https://www.frontiersin.org/articles/10.3389/fncir.2019.00022/full), enabling similar principles to be applied to image-based tasks. 
+The following implements a network using grid-cell computations that can perform basic object recognition on visual tasks such as the MNIST data-set. It builds on the architecture described in [Lewis et al 2019](https://www.frontiersin.org/articles/10.3389/fncir.2019.00022/full), enabling similar principles to be applied to image-based tasks. The repository can be used to generate the results found in our BMVC 2021 paper, "Grid Cell Path Integration For Movement-Based Visual Object Recognition".
```

Comment:
Maybe link to the paper in this README as well? We could just add a URL to the title.